test/Microsoft.SqlTools.ServiceLayer.TestDriver/Utility/TestRunner.cs

```diff
@@ -23,9 +23,8 @@ public class TestRunner
                     bool containsTestName = testName.Contains(".");
                     var className = containsTestName ? testName.Substring(0, testName.LastIndexOf('.')) : testName;
                     var methodName = containsTestName ? testName.Substring(testName.LastIndexOf('.') + 1) : null;
-
-
-                    var type = Type.GetType(testNamespace + className);
+                    Assembly assembly = Assembly.GetEntryAssembly();
```

Comment:
I think rather than getting the entry assembly, we should get the assembly of the current type, ie TestRunner.GetType().Assembly (or similar). The issue I foresee happening is that the entry assembly won't be the what we expect when we run it from Test Explorer